samples/ingestion/ingestion-client/Setup/guide.md

```diff
@@ -16,7 +16,7 @@ The following diagram shows the structure of this tool as defined by the ARM tem
 
 When a file lands in a storage container, the Grid event indicates the completed upload of a file. The file is filtered and pushed to a Service bus topic. Code in Azure Functions triggered by a timer picks up the event and creates a transcription request using the Azure Speech services batch pipeline. When the transcription request is complete, an event is placed in another queue in the same service bus resource. A different Azure Function triggered by the completion event starts monitoring transcription completion status. When transcription completes, the Azure Function copies the transcript into the same container where the audio file was obtained.
```

Comment:
Maybe create a new Issue to move this setup folder into `Infra` - so when you add `azd` its all tidied up. Feel free to resolve this comment when you've raised the issue.

Reply:
I have created the issue in our board, so that we can track all items related to the bicep template at one place.
https://github.com/orgs/microsoft/projects/1200/views/1?pane=issue&itemId=68857172
Let me know your thoughts